packages/integration-tests/index.js

```diff
@@ -564,6 +563,46 @@ async function deleteRules(stackName, bucketName, rules, postfix) {
   return rules.length;
 }
 
+/**
+ * Delete a rule's Kinesis Event Source Mappings
+ *
+ * @param {Object} rule - a Rule record as returned by the Rules api
+ * @param {string} rule.name
+ * @param {Object} rule.rule
+ * @param {string} rule.rule.arn
+ * @param {string} rule.rule.logEventArn
+ * @returns {Promise<Object>[]} - Event Source Map deletion results
```

Comment:
1. Subtle difference, this is a Promise that resolves to an array of things, as opposed to an array of Promises that each resolve to a single value.
2. Let's use `unknown` here instead of `Object`.

```suggestion
 * @returns {Promise<unknown[]>} - Event Source Map deletion results
```